mne/io/boxy/boxy.py

```diff
@@ -18,7 +18,7 @@
 
 
 @fill_doc
-def read_raw_boxy(fname, preload=False, verbose=None):
+def read_raw_boxy(fname, datatype=None, preload=False, verbose=None):
```

Comment:
add in comments here about this parameter and what its possible values can be, does it take strings, does it take lists of strings? Also choose a better default value (probably AC) by replacing None with 'AC'

Reply:
okay, I'll add a brief description

Reply:
click resolve once you do and push the change